packages/swap/src/translators/reactjs/state/state.ts

```diff
@@ -606,7 +600,7 @@ const orderReducer = (
         draft.orderData.calculations = makeOrderCalculations({
           orderType: state.orderData.type,
           amounts: state.orderData.amounts,
-          limitPrice: state.orderData.limitPrice,
+          limitPrice: undefined,
```

Comment:
if limit is passed it won't get updated, the if (shouldn't happen) the FFEE changes while creating an order, it must be reseted to market